cl/api/tests.py

```diff
@@ -3456,3 +3463,158 @@ def test_anonymous_user_handling(self, mock_get_redis):
         dates = list(anonymous_data.keys())
         dates.remove("total")
         self.assertEqual(dates, ["2023-01-01", "2023-01-02"])
+
+
+@patch("cl.api.utils.get_cache_key_for_no_filter_mixin")
+class CacheListApiResponseTest(TestCase):
+    def setUp(self):
+        self.cache = caches["db_cache"]
```

Comment:
Why not use redis cache during tests for speed?

Reply:
oh that's interesting, I'm just setting up a variable to avoid repetition in the tests. The cache connection within the mixin is not configurable and is fixed to the database cache.